### PR TITLE
Add Intel H2312XXKR2 Blade Chassis

### PR DESCRIPTION
**Pull request by getspaghett**
Added Intel H2312XXKR2 Blade Chassis definition with specifications taken from [intel.com](https://www.intel.com/content/www/us/en/products/sku/81877/intel-server-chassis-h2312xxkr2/specifications.html)

### CODE REVIEW

File: device-types/Intel/H2312XXKR2.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+manufacturer: Intel
+model: H2312XXKR2
+slug: intel-h2312xxkr2
+u_height: 2
+is_full_depth: true
+comments: |
+  [H2312XXKR2 Specifications on intel.com](https://www.intel.com/content/www/us/en/products/sku/81877/intel-server-chassis-h2312xxkr2/specifications.html)
+subdevice_role: parent
```
Comment (danner26) [inline device-types/Intel/H2312XXKR2.yaml:9]: Please add `airflow` if you have it

File: device-types/Intel/H2312XXKR2.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+device-bays:
+  - name: Node 1
+    label: Blade 1
+  - name: Node 2
+    label: Blade 2
+  - name: Node 3
+    label: Blade 3
+  - name: Node 4
+    label: Blade 4
+power-ports:
```
Comment (danner26) [inline device-types/Intel/H2312XXKR2.yaml:19]: Just as in the other one, the PSUs are modular. This definition should be removed and `module-bays` should be created
Then create the applicable PSUs in the module-types folder

Looks like the PSUs are `FXX1600PCRPS`

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Hello, unfortunately today we had to make a change to the schema in which weight is defined by. The upstream NetBox API requires weight to be passed to it in the following format:
```
weight: 12.01
weight_unit: lb
```

Due to this change, I have updated your branch against the current master. This might require you to update your PR, if you had weight currently defined, so that your PR is in-line with the new weight schema. If you have any issues, please let me know or refer to [this PR](https://github.com/netbox-community/devicetype-library/pull/1187) for more information.

**Comment by danner26** [comment]
Anything else needed for this PR?

**Comment by getspaghett** [comment]
> Anything else needed for this PR?

Nope, i think it is ready to be merged :) :+1:

**Review by danner26 — APPROVED** [review]

**Pull request merged**